packages/plugins/test/mocha/dummy.ts

```diff
@@ -285,3 +306,37 @@ export const getDummyDeploymentArtifacts = (): DeploymentArtifacts => {
     },
   }
 }
+
+export const getDummyDeploymentContext = (): DeploymentContext => {
```

Comment:
Ended up not using this, but I kept it in b/c I figured it'll probably be useful at some point soon